### ISSUE TITLE
EmailSignupTemplate doesn't have a clear purpose

### ISSUE THREAD

**Issue by lyudmil** [classicrob/GuidedTrack-Templates#1]
The email that the program ends up sending refers to an inquiry that the user can expect a follow-up email about, but the user is never asked to enter a query. It seems like we're stuck in-between showing users how to send emails and how to sign up for a newsletter via Mailchimp. The latter doesn't seem like something templates are well-suited to teach, and the implementation of the former is incomplete.

I suggest we implement a "contact me" form that ends up sending a question the user typed to the author *and* a confirmation email to the participant. That way the focus of the template becomes `*email`, and the Mailchimp dimension can remain a comment that urges the user to explore further.

**Comment by classicrob** [comment]
Yeah we can remove the Mailchimp stuff. Wanna adjust it in the way you see fit?

Quick note about the template:

I intentionally have some questions on a page together and some not to demonstrate the difference and how questions normally end the pages. I'd like to keep that part in place.
